community-content/prediction_featurestore_integration/prediction_featurestore_integration.ipynb

```diff
@@ -0,0 +1,1470 @@
+{
```

Comment:
<p>Consider writing out what the final config looks like.</p><p> </p><p>Also consider include passthrough fields for demonstration purposes.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/1371/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='62'/>

Reply:
The change to enable passthrough fields is not submitted yet.